src/commands/transaction/accountmosaicrestriction.ts

```diff
@@ -81,6 +81,9 @@ export default class extends AnnounceTransactionsCommand {
             'Introduce the maximum fee (absolute amount): ');
 
         const profile = this.getProfile(options);
+        if (this.aliasTag === options.value) {
```

Comment:
We could use ``MosaicService.getRecipient(options.value);``

Reply:
fixed